AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3302,7 +3302,7 @@ duckduckgo.com##.js-badge-cookie-msg
 mall.pl##.js-consent-panel
 lingvanex.com,home.pl,samsung.com##.js-cookie
 ing.pl##.js-cookie-policy-box
-blog.twitter.com##.js-eu-cookie-notice
+mopub.com,blog.twitter.com##.js-eu-cookie-notice
```

Comment:
It's covered by `mopub.com,twitter.com##.u12-data-protection-notice`.

Reply:
OKay.